config/features.yml

```diff
@@ -388,4 +393,4 @@ features:
     actor_type: user
     enable_in_development: true
     description: >
-       Include state abbreviation when searching on comparison tool 
\ No newline at end of file
+      Include state abbreviation when searching on comparison tool
```

Comment:
Was the change meant to be included?

Reply:
My IDE must have automatically made that adjustment.  I will revert.

Reply:
Fixed!

Reply:
Awesome, thanks!